recipes/coremltools/build.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -e
+set -x
+mkdir build
+cd build
+
+cmake \
+    -DCMAKE_OSX_DEPLOYMENT_TARGET=10.13 \
```

Comment:
One minor thing and then it looks good once the other maintainers confirm:

```suggestion
    -DCMAKE_OSX_DEPLOYMENT_TARGET=${MACOSX_DEPLOYMENT_TARGET} \
```

Reply:
Good catch!